presto-kafka/src/test/java/io/prestosql/plugin/kafka/TestKafkaSecurityConfig.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.kafka;
+
+import io.prestosql.testing.assertions.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Properties;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestKafkaSecurityConfig
+{
+    @Test
```

Comment:
Extend the test with common test methods used in all `*Config` classess tests:
 * `testDefaults`
 * `testExplicitPropertyMappings`

Reply:
done